packages/windows/src/RPA/Windows/__init__.py

```diff
@@ -485,7 +485,7 @@ def __init__(self):
         libraries = [
             ActionKeywords(self),
             ElementKeywords(self),
-            LocatorKeywords(self),
+            LocatorKeywords(self, kwargs.get("locators_path", None)),
```

Comment:
```suggestion
            LocatorKeywords(self, **kwargs),
```
can be more generic if you like it.